pkg/cluster/kubeclient/ready.go

```diff
@@ -56,6 +56,9 @@ func (u *kubeclient) WaitForReadySyncPod(ctx context.Context) error {
 					u.log.Info(err.ErrStatus.Details.Causes[0].Message)
 				}
 				return false, nil
+			case ready.IsNetworkUnreachableError(err):
```

Comment:
did you see this happen? should be dealt with 1 layer down in the kubeclient retryer